compiler/plugins/target/AMD-AIE/aie/test/aiert_insts.mlir

```diff
@@ -5,7 +5,7 @@
 // CHECK:           memref.global "public" @of_fromMem : memref<32xi32>
 // CHECK:           aie.shim_dma_allocation @of_fromMem(MM2S, 0, 0)
 // CHECK:           aie.shim_dma_allocation @of_toMem(S2MM, 0, 0)
-// CHECK:         } {npu_instructions = dense_resource<npu_instructions> : tensor<64xui32>}
+// CHECK:         } {npu_instructions = dense_resource<npu_instructions> : tensor<64xui32>, runtime_sequence_name = "sequence"}
```

Comment:
not 100% clear on why we need this? In case there are multiple sequence ops with different names?

Reply:
This is used used for the case of multiple aie.device ops in a module (seems like a slightly more plausible situation than multiple aiex.runtime_sequences in an aie.device). Because of the reordering of passes in this PR, the aiex.runtime_sequence is gone by the time the code here: https://github.com/nod-ai/iree-amd-aie/blob/d1073056ffa47b295da29c4df3ab0618c2e4f825/compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/AIETarget.cpp#L97 needs to get the function name. So I've transplanted it onto the device op.

Reply:
Oh I get it (sorry it's late and I'm underslept). Well btw since we now own these ops you can just add it as a real actual attribute on `DeviceOp`.